tools/tikv-control.md

```diff
@@ -99,6 +105,28 @@ In this command, the key is also the escaped form of raw key.
 
 To print the value of a key, use the `print` command.
 
+### Print some properties about region
+
+In order to record region state details, `TiKV` writes some statistic into SST files of regions.
+For take a look at these properties, run `tikv-ctl` with sub-command `region-properties`:
+
+```bash
+$ tikv-ctl --host localhost:20160 region-properties -r 2
+num_files: 0
+num_entries: 0
+num_deletes: 0
+mvcc.min_ts: 18446744073709551615
+mvcc.max_ts: 0
+mvcc.num_rows: 0
+mvcc.num_puts: 0
+mvcc.num_versions: 0
+mvcc.max_row_versions: 0
+middle_key_by_approximate_size:
+```
+
+The properties can be used for tell the region is health or not. If not, then we can use them to fix the region.
+E.g. split the region manually by the `middle_key_approximate_size`.
```

Comment:
Update these two lines to:

The properties can be used **to check whether the Region is healthy** or not. If not, **you can** use them to fix the **Region**. For example, splitting the **Region** manually by ~~the~~ `middle_key_approximate_size`.